docs/readmes/orc8r/dev_minikube.md

```diff
@@ -45,6 +45,7 @@ There are 2 ways you can publish your own images: to a private registry, or to a
     - `minikube addons enable registry`
     - `docker run --rm -it --network=host alpine ash -c "apk add socat && socat TCP-LISTEN:5000,reuseaddr,fork TCP:$(minikube ip):5000"`
     - Use `localhost:5000` as your registry name
+    - This should hang, open new tab for other commands
```

Comment:
```suggestion
    - `docker run --rm -it --network=host alpine ash -c "apk add socat && socat TCP-LISTEN:5000,reuseaddr,fork TCP:$(minikube ip):5000"` (this should hang, open new tab for other commands)
    - Use `localhost:5000` as your registry name
```

Reply:
I feel like it's easier for ppl to copy paste if it's on a new line? But I agree it should probably be before the 
```
    - Use `localhost:5000` as your registry name
```
? Thoughts?